types/index.d.ts

```diff
@@ -2205,7 +2207,16 @@ export declare namespace Knex {
     queryContext(context: any): ColumnBuilder;
     after(columnName: string): ColumnBuilder;
     first(): ColumnBuilder;
+    check(checkPredicate: string, bindings?: any, constraintName?: string): ColumnBuilder;
+    checkPositive(constraintName?: string): ColumnBuilder;
+    checkNegative(constraintName?: string): ColumnBuilder;
+    checkIn(values: string[], constraintName?: string): ColumnBuilder;
+    checkNotIn(values: string[], constraintName?: string): ColumnBuilder;
+    checkBetween(values: any[] | any[][], constraintName?: string): ColumnBuilder;
+    checkLength(operator: string, length: number, constraintName?: string): ColumnBuilder;
```

Comment:
is list of supported operators DB-specific? wonder if it would be feasible to narrowly type all the options